src/api/app/controllers/webui/staging/workflows_controller.rb

```diff
@@ -35,8 +35,8 @@ def show
     @more_unassigned_requests = @staging_workflow.unassigned_requests.count - @unassigned_requests.size
     @ready_requests = @staging_workflow.ready_requests.first(5)
     @more_ready_requests = @staging_workflow.ready_requests.count - @ready_requests.size
-    @ignored_requests = @staging_workflow.ignored_requests.first(5)
-    @more_ignored_requests = @staging_workflow.ignored_requests.count - @ignored_requests.size
+    @excluded_requests = @staging_workflow.excluded_requests.first(5)
+    @more_excluded_requests = @staging_workflow.excluded_requests.count - @excluded_requests.size
```

Comment:
Use [offset](https://guides.rubyonrails.org/active_record_querying.html#limit-and-offset) here:wink:

Reply:
I would suggest that we don't refactor code that was already there in this PR.

Anyway, how do you propose to use `offset` in this line? I don't think `offset` can be use here... :confused:

Reply:
Nevermind, I [confused](https://github.com/openSUSE/open-build-service/pull/6201#discussion_r234602337) this with something else.